x-pack/legacy/plugins/rollup/public/legacy.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  npSetup,
+  npStart,
+  aggTypeFilters,
+  aggTypeFieldFilters,
+  editorConfigProviders,
+} from './legacy_imports';
+import { addSearchStrategy } from '../../../../../src/legacy/core_plugins/data/public/search/search_strategy';
+import { RollupPlugin } from './plugin';
+import { setup as management } from '../../../../../src/legacy/core_plugins/management/public/legacy';
+import { addBadgeExtension, addToggleExtension } from '../../index_management/public';
+
+const plugin = new RollupPlugin();
+
+export const setup = plugin.setup(npSetup.core, {
```

Comment:
That's the only place `npSetup`/`npStart` is allowed to be used, but it should be imported directly via
```
import { npSetup, npStart } from 'ui/new_platform';
```